pkg/controller/node/nodecontroller.go

```diff
@@ -63,6 +64,18 @@ var (
 	// The minimum kubelet version for which the nodecontroller
 	// can safely flip pod.Status to NotReady.
 	podStatusReconciliationVersion = utilversion.MustParseSemantic("v1.2.0")
+
+	UnresponsiveTaintTemplate = &v1.Taint{
+		Key:    metav1.TaintNodeUnreachable,
+		Value:  "NodeControllerTaint",
+		Effect: v1.TaintEffectNoExecute,
+	}
+
+	NotReadyTaintTemplate = &v1.Taint{
+		Key:    metav1.TaintNodeNotReady,
+		Value:  "NodeControllerTaint",
```

Comment:
Please remove the Value in both of the templates. For example if we change not-ready to be set by kubelet, it is easier if pods who already tolerate them don't need to change the toleration.